[
  {
    "signature": "error(java.lang.String msg, java.lang.Exception ex, int code)",
    "name": "error",
    "containingClass": {
      "qualifiedName": "java.util.logging.ErrorManager",
      "name": "ErrorManager",
      "isArray": false
    },
    "targetClass": "java.util.logging.ErrorManager",
    "isVarArgs": false,
    "returnType": {
      "qualifiedName": "void",
      "name": "void",
      "isArray": false
    },
    "parameters": [
      {
        "type": {
          "qualifiedName": "java.lang.String",
          "name": "String",
          "isArray": false
        },
        "name": "msg"
      },
      {
        "type": {
          "qualifiedName": "java.lang.Exception",
          "name": "Exception",
          "isArray": false
        },
        "name": "ex"
      },
      {
        "type": {
          "qualifiedName": "int",
          "name": "int",
          "isArray": false
        },
        "name": "code"
      }
    ],
    "protocol": {
      "member": "error(java.lang.String msg, java.lang.Exception ex, int code)",
      "comment": "The error method is called when a Handler failure occurs.  This method may be overridden in subclasses. The default behavior in this base class is that the first call is reported to method_0, and subsequent calls are ignored.",
      "kind": "FreeText",
      "condition": ""
    }
  }
]